### ISSUE TITLE
belyi_db_improved

### ISSUE THREAD

**Issue by jvoight** [michaelmusty/BelyiDB#19]
What the "improved" mean?  How is it different than belyi_db?  Which one is being used when we upload maps?

**Comment by SamSchiavone** [comment]
I believe belyi_db_improved has files with improved equations for genus 0 maps that Ciaran @cscmnu added after our virtual Belyi LMFDB mini-meeting.

**Comment by jvoight** [comment]
So they're not currently part of the upload script?

**Comment by SamSchiavone** [comment]
Right, they're not part of the upload, and yes, I think we can delete them, although you might want to double-check with Ciaran.

**Comment by cscmnu** [comment]
Yes please delete them, there's nothing in there that we haven't superseded with Gm-Reduce
